test/it/006activityLogTest.js

```diff
@@ -18,7 +18,7 @@ describe('Activity Log Test', async () => {
 
   it('Activity Log on Survey Creation', async () => {
     const surveyParam = {
-      name: 'test_survey_' + uuidv4(),
+      name: 'DO NOT USE! test_survey_' + uuidv4(),
```

Comment:
I would have used a "normalized" name in this case, in the way we normalize internal names (lowercase, without spaces) because the survey name should be like that.
During tests the surveys are published: we don't validate the survey name on survey publish because it has been already validated on survey creation and on survey edit, but we could check it in theory.

Reply:
Ah, I didn't notice that. I guess it's the labels that are meant to be the human-readable names (and the names likes this I guess are mostly hidden). I'll correct that.